modules/administration/pages/ssl-certs-hsts.adoc

```diff
@@ -0,0 +1,12 @@
+[[ssl-certs-hsts]]
+= HTTP Strict Transport Security
+
+For additional security with regard to certificates and better protection against specific MITM attacks, it is possible to enable [HTTP Strict-Transport-Security](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Strict-Transport-Security).
+This is done by uncommenting the following line in the Apache Webserver configuration on the {productname} server as well as {productname} proxies if used:
```

Comment:
To enable Strict Transport Security uncomment the following line in the Apache Webserver configuration on the {productname} server and {productname} proxies.